.travis.yml

```diff
@@ -44,6 +55,8 @@ before_install:
     esac
   - echo -n | openssl s_client -connect scan.coverity.com:443 | sed -ne '/-BEGIN CERTIFICATE-/,/-END CERTIFICATE-/p' | sudo tee -a /etc/ssl/certs/ca-
 
+before_script:
```

Comment:
I had to look twice here. You essentially download a specific version of `go`. There two things that bother me about this:

1. I don't like downloading random stuff when doing CI. I guess you could argue this is not random stuff and it is needed to generate the display names as you found out
2. if we really want this then shouldn't this go in `.travis/before_install/gcc.sh`?
3. I guess it boils down to the fact that I never liked the dependency on `go` in the first place, but that is not your fault

Reply:
Thanks for feedback!!

Go is installed manually because the job was failing with "go command not found" for ppc64le  and since its not a good idea to download packages randomly, I will try to configure default go package which is present inside the ppc64le base image.  And i have to figure it out that why it's not taking the default one which present inside the base package.

Thanks !!

Reply:
Hi @egli , I debuged it and concluded that the existing 'go' package is not suitable for multi-architecture env. 
So it would be better if you could please update the 'go' package of base image for multi-arch support.

Or,

I can update the code and will remove the 'MODE' where 'go' is failing. and will remove the check-format MODE as well.
Just there will be only one MODE for ppc64le.  :)

Please confirm me on this.
Thanks !!